pod/perlop.pod

```diff
@@ -2989,91 +3055,82 @@ delimiter) will be preserved:
                 This text is indented with two tabs
```

Comment:
The example claims that it is indented with two tabs, while it is indented with 10 spaces. However, I am unsure how important this is, as tabs might not survive the conversion from pod to different representations, e.g. HTML.

Reply:
I made sure it really is 2 tabs.  Please check  if that survived for you

Reply:
Thank you. The tabs survive in the pod file, but they do get turned into spaces when I convert them to .man or .html. I an unsure how the conversion is carried out in the specific Perl website pipelines, but I imagine the result would be similar (this is probably one of the few rare cases where tabs actually matter). Nevertheless, at least the example in the pod file is correct now.

Reply:
Nice idea, thanks.